### ISSUE TITLE
Dropdown for selecting programming club

### ISSUE THREAD

**Issue by zachlatta** [hackedu/shipit#6]
Create a dropdown to choose your programming club from a list. When it's a limiting factor, this dropdown can also have search.

**Comment by MaxWofford** [comment]
It makes sense to me. Is there a centralized place to scrape/get all hackEDU club names?

**Comment by zachlatta** [comment]
Yeah, we sent out a survey to all of the clubs last week and should have
the data by now.

I can send you a list of the names if you'd like to implement it.

**Comment by MaxWofford** [comment]
Do we have that data on a server? It would be great to populate that list from real-time club data.

If not, go ahead and PM me the list of names and I will implement it.

**Comment by zachlatta** [comment]
Nowhere right now :/

Totally agree that it'd be ideal if we got it from the actual club data.
We'll have to eventually expose an API from hackedu/hackedu.

I'll reply to this thread with the club names in the meantime.

**Comment by zachlatta** [comment]
Schools are as follows:

```
Austin High School
Bronx High School of Science
Cherry Hill High School East
Cranbrook Schools
El Segundo High School
Elizabeth High School
Gig Harbor High School
Gunn High School
International Academy East
Lower Moreland High School
Middlesex County Academy for Science, Math, and Engineering Technologies
Mission Vista High School
Mundelein High School
Radnor High School
Ravenwood High School
San Mateo High School
Santa Teresa High School
South Brunswick High School
The Shri Ram School - DLF Campus
Thunderridge High School
Townsend Harris High School
Twin Cities Academy
West Orange High School
```

Let's just go with school names instead of club names. I'll go ahead and rename the ESHS Programming Club to El Segundo High School.

**Comment by zachlatta** [comment]
Closed by #23.

**Comment by MaxWofford** [comment]
The dropdown doesn't submit anything to the form.

**Comment by zachlatta** [comment]
It'd be great if you could do this ASAP.

**Comment by MaxWofford** [comment]
This should be fixed by PR #29

**Comment by zachlatta** [comment]
Cool, merged.

--
Zach Latta http://zachlatta.com https://github.com/zachlatta
https://twitter.com/zachlatta


On Tue, Jan 27, 2015, at 06:32 PM, Max Wofford wrote:
> This should be fixed by PR #29[1]


> — Reply to this email directly or view it on GitHub[2].

>



Links:

  1. https://github.com/hackedu/shipit/pull/29
  2. https://github.com/hackedu/shipit/issues/6#issuecomment-71770876
